packages/lit-dev-content/site/tutorials/content/async-directive/05.md

```diff
@@ -0,0 +1,77 @@
+To make our directive update asynchronously, we'll need to start a timer that
+re-renders the time elapsed.
+
+Add a function that starts a periodic 1s timer if it's not already started.
+Since directives are stateful, we can use a class field to store the timer's
+handle, and use that to start the timer only once:
+
+{% switchable-sample %}
+
+```ts
+  timer: number | undefined;
+
+  ensureTimerStarted() {
+    if (this.timer === undefined) {
+      this.timer = setInterval(() => {
+        /* do some periodic work */
+      }, 1000);
```

Comment:
```suggestion
      }, 3000);
```

As previously mentioned, I found the 1 second updates really distracting/confusing.